app/models/foreman_openscap/policy.rb

```diff
@@ -21,7 +21,7 @@ class Policy < ActiveRecord::Base
     SERVER_CLASS_PARAMETER   = 'server'
     PORT_CLASS_PARAMETER     = 'port'
 
-    validates :name, :presence => true, :uniqueness => true, :format => { :without => /\s/ }
+    validates :name, :presence => true, :uniqueness => true
```

Comment:
Not as far as I know. And we use this name only to display it on foreman.
Maybe in the past it was used to set the puppet? We switched to id long time ago though